feat: show info message if Herodotus voting power is not available yet on a proposal

Sekhmet opened the pull request:
### Summary

This PR updates Herodotus to use new indexer API and display message if Herodotus is not processed yet.

Closes: https://github.com/snapshot-labs/sx-monorepo/issues/154
Closes: https://github.com/snapshot-labs/sx-monorepo/issues/207

**Note: Currently new indexer has CORS issue so all requests fail showing this message, waiting for fix from Herodotus.**

### How to test

1. Run `yarn dev:full` (might be good idea not to run subgraph and set Sepolia start block later to prevent rate limits).
2. Create space with ozvotes/evmslotvalue strategy (can you this token: `0x6fd821e79cdf212ad8b06c59b28fe8c2185291d4`, slot index 0 for EVM slot value, 8 for OZ Votes).
3. Create proposal.
4. It shows this message until confirmed.

### Screenshots

Newer design:
<img width="361" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/933666bb-babe-4c55-bd4d-80295d7f6af9">


New design:
<img width="930" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/c63121ff-7559-4554-9af8-c9c25fb3578c">


Previous design:
<img width="992" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/46dadd0c-fe6d-41be-a31a-cd350e37af32">

bonustrack commented:
Can you change the style of the message, it's too much highlighted current I recommend something like this: 
<img width="601" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/df185d64-8acd-4eef-b847-8aadac696cb5">

Sekhmet commented:
@bonustrack updated, I removed icon as text always ends up in 3 lines and it looks pretty bad in that case. Only message is left I changed color to use default as without icon it looked confusing with both headline and message in white.

bonustrack commented on apps/ui/src/views/Proposal.vue:
```diff
@@ -154,7 +163,16 @@ watchEffect(() => {
+              "
+              class="mt-2"
+            >
+              Please allow few minutes for the voting power to be collected from Ethereum.
```
Can you add the icon "information-circle" here too?

bonustrack replied:
Ok sorry just seen your comment. Best would be to show Icon inline with the text and not taking 3 line.

Sekhmet replied:
You mean that icon should align to first line (because there will always be 3 lines because there is not enough space). I tried it before but it didn't look great too.

I can update the designs again to see how it looks like.

bonustrack replied:
Something like this
<img width="317" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/4f67264e-ea9d-4972-8ba6-3e3483e9e128">

bonustrack replied:
There is the same issue on empty messages across the app
<img width="215" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/18eb4b1a-4d5a-4ca3-97d3-aba6a09affbe">

bonustrack commented:
I can't see the message when I test
<img width="1469" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/16245250/c90e70ab-4813-4d30-8776-04faba7bf900">
Here: http://localhost:8080/#/sn:0x0041ada9121061198b52ae28edeec8ace7c23f2ba8d66938c129af1a2245701c/proposal/15

Sekhmet commented:
@bonustrack updated, looks like Herodotus updated their API format.

bonustrack reviewed (approved):
tACK

Sekhmet commented:
Updated for the new UI, not sure if it's what we wanted, it looks like this now (this message replaces `Your voting power` message, so it's similar than before, but not those two sections are merged so not sure if it's good:

<img width="397" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/f67dff96-682c-4d8b-89db-e8099283fa42">
